File: app/src/main/java/com/weathercock/cleanarchitecture/presentation/utils/ViewBindingUtils.java
package com.weathercock.cleanarchitecture.presentation.utils;

import android.databinding.BindingAdapter;
import android.graphics.drawable.Drawable;
import android.widget.ImageView;

import com.squareup.picasso.Picasso;

public class ViewBindingUtils {

    @BindingAdapter({"imageUrl"})
    public static void loadImage(ImageView view, String url) {
        Picasso.with(view.getContext()).load(url).into(view);
    }

    @BindingAdapter({"imageUrl", "error"})
    public static void loadImage(ImageView view, String url, Drawable error) {
        Picasso.with(view.getContext()).load(url).error(error).into(view);
    }

}

File: app/src/main/java/com/kazakago/cleanarchitecture/presentation/utils/ViewBindingUtils.kt
package com.kazakago.cleanarchitecture.presentation.utils

import android.databinding.BindingAdapter
import android.graphics.drawable.Drawable
import android.widget.ImageView

import com.squareup.picasso.Picasso

@BindingAdapter("imageUrl")
fun ImageView.imageUrl(url: String) {
    Picasso.with(this.context).load(url).into(this)
}

@BindingAdapter("imageUrl", "error")
fun ImageView.imageUrl(url: String, error: Drawable) {
    Picasso.with(this.context).load(url).error(error).into(this)
}